docs/2.9.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/overview.rst

```diff
@@ -6,17 +6,17 @@ Overview
 
 **High Availability** (HA) is the elimination of single points of failure to ensure that applications continue to operate when a component they depend on, such as a server, fails. 
 
-HA for Daml solutions focuses on the following components running in separate processes:
+HA for Daml solutions focus on the following components running in separate processes:
```

Comment:
Revert -- the subject is "HA" so "focuses" is correct. 

And I think you just proved that the awkward phrasing makes the sentence confusing. Could rephrase to "In Daml solutions, HA focuses..." (still pretty awkward)